src/earn/PoolCard.tsx

```diff
@@ -92,6 +95,14 @@ export default function PoolCard({ pool, testID = 'PoolCard' }: { pool: Pool; te
           <View style={styles.buttonContainer}>
             <Button
               onPress={() => {
+                ValoraAnalytics.track(EarnEvents.earn_pool_card_cta_press, {
+                  poolId: pool.poolId,
+                  depositTokenId,
+                  networkId: pool.networkId,
+                  tokenAmount: poolTokenInfo.balance.toString(),
+                  providerId: PROVIDER_ID,
```

Comment:
leave a note that this should be generalized. Pretty easy to miss when adding more pools. Or alternatively, change this to a more common string type in the analytics properties and get this from the pool config file for now (which can eventually be replaced by the hooks endpoint)

Reply:
added TODOs linking to the analytics update follow-up ticket. Currently providerId isn't part of the `Pool` object, and so would have to update that object which would result in a lot of test files needing to be changed so left it for the follow-up ticket to make this cleaner.